src/Modules/Backend/Domain/ModuleSettings/ModuleSettingsType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Domain\ModuleSettings;
+
+use ForkCMS\Core\Domain\Form\SwitchType;
+use ForkCMS\Modules\Extensions\Domain\Module\Command\ChangeModuleSettings;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Expression;
+
+final class ModuleSettingsType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add(
+                '2fa_enabled',
+                CheckboxType::class,
+                [
+                    'required' => false,
+                    'label' => 'lbl.EnableTwoFactorAuthentication',
```

Comment:
Can you use `TranslationKey::label('EnableTwoFactorAuthentication')` instead
I would like to avoid using translation keys as just strings as much as possible in the code.
I have some places in my code where I would have to refactor it as well but might be nicer to not add new instances that need refactoring

Reply:
fixed fb0a5f1e048250399f7cb1eae5c12faed8861a3c